### DISCUSSION TITLE
Rewrite it with pure Rust?

### DISCUSSION THREAD

**Discussion by Unknown78** [Alex313031/Thorium-Win#177] [Ideas]
According to [openhub](https://openhub.net/p/chrome/analyses/latest/languages_summary), only 2.4% is in Rust.

"The fastest browser on Earth."
We don't want only to be fast, but also robust.
Why don't go one step further?

**Comment by gz83** [comment]
The Chromium browser and its derivative projects are quite complex and large.  It is up to Google to decide whether to rewrite the browser's code, and it does not currently do so.
